apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -376,6 +376,7 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
             <FileInputButton
               data-testid="manual-input"
               onChange={processCastVoteRecordFileFromFilePicker}
+              accept=".json"
```

Comment:
There are ways to allow the user to select directories, but I think they're not intuitive to the user (and require some annoying React HTML workarounds), so let's just let them select the `.json` report and handle the path on the backend.

Reply:
Cute workaround